src/main/kotlin/no/nav/dagpenger/regel/api/db/PostgresBruktSubsumsjonStore.kt

```diff
@@ -56,14 +61,14 @@ class PostgresBruktSubsumsjonStore(private val dataSource: HikariDataSource) : B
             return using(sessionOf(dataSource)) { session ->
                 session.run(
                     queryOf(
-                        """SELECT * FROM v1_subsumsjon_brukt" +
+                        """SELECT * FROM v1_subsumsjon_brukt
                         WHERE id = ?""".trimMargin(), subsumsjonsId
                     ).map { row ->
                         SubsumsjonBrukt(
                             id = row.string("id"),
                             eksternId = row.string("ekstern_id"),
-                            arenaTs = row.string("arena_ts"),
-                            ts = row.long("created")
+                            arenaTs = row.zonedDateTime("arena_ts").format(secondGranularityFormatter),
```

Comment:
Det lukter litt at vi burde tenke litt datatyper her. Arena/GG poster tidspunktet som en string med mikrosekund oppløsning. Vi tenkte opprinnelig at timestamp with time zone var riktig her også, men mulig output burde representere det med riktige typer istedet for POST bodyen fra http requesten

Reply:
Og vi har vel enda ingen brukere av getBruktSubsumsjon, slik at det fortsatt ikke er for sent å flytte litt på det, så vi slipper å konvertere oss til stringly typed felter

Reply:
Fikset i 6c649ff